hardware_interface:
  steps_per_m: 250
  dirs:
    - 88
    - -1
    - -1 
    - -1 
    - -1 
    - -1 
  steps:
    - 87
    - -1 
    - -1 
    - -1 
    - -1 
    - -1 
  joints:
    - mid_left_wheel_joint
    - mid_right_wheel_joint
    - front_left_wheel_joint
    - front_right_wheel_joint
    - back_left_wheel_joint
    - back_right_wheel_joint
